src/main/java/frc/robot/Robot.java

```diff
@@ -15,6 +17,11 @@
  * project.
  */
 public class Robot extends TimedRobot {
+
+  public static DriveTrain driveTrain;
```

Comment:
you never initialize `DriveTrain`. instead you init it in as `m_exampleSubsystem` in the `RobotContainer`